tests/symengine/symengine_tensor_operations_04.cc

```diff
@@ -132,7 +140,7 @@ test_symmetric_tensor()
   Tensor_t t_a, t_b;
   for (unsigned int i = 0; i < Tensor_t::n_independent_components; ++i)
     {
-      const auto index = t_a.unrolled_to_component_index(i);
+      const auto index = t_a.unrolled_to_component_indices(i);
```

Comment:
Oh, did that ever work?

Reply:
No, with this typo this never this never compiled.